clang/lib/Driver/Driver.cpp

```diff
@@ -733,8 +733,22 @@ void Driver::CreateOffloadingDeviceToolChains(Compilation &C,
           C.getInputArgs().getLastArg(options::OPT_fsycl_link_targets_EQ);
   Arg *SYCLAddTargets =
           C.getInputArgs().getLastArg(options::OPT_fsycl_add_targets_EQ);
+  bool SomeSYCLTargetsOption = SYCLTargets || SYCLLinkTargets || SYCLAddTargets;
```

Comment:
Would HasSYCLTargetOptions or HasSYCLTargetsOptions be a better name?

Reply:
I will rename it into HasSYCLTargetsOptions.

Reply:
Done

Reply:
Thanks! Awesome work.